package.json

```diff
@@ -84,5 +84,10 @@
     "Matt Dobson <mdobson@apigee.com> (https://github.com/mdobson)",
     "Mitch Fierro <fierro.mitch@gmail.com> (https://github.com/f1erro)"
   ],
-  "author": "Kevin Swiber <kswiber@gmail.com>"
+  "author": "Kevin Swiber <kswiber@gmail.com>",
+  "engines" : { 
+     "node" : ">= 4.8.2 <=6.9.1",
```

Comment:
Maybe? I remember an internal conversation where we wanted to bring the restriction forward. Not married to it, just raising the issue.